doc/ref/chap-type-method.md

```diff
@@ -38,7 +38,7 @@ See [sh-array][] for details.  In YSH, prefer to use [List](#List) instances.
 
 A bash associative array is a mapping from strings to strings.
 
-See [sh-assoc][] for details.  In YSH, prefer to use [Dict](#Dict) instances.
+See [sh-assoc][] for the initialization.  In YSH, prefer to use [Dict](#Dict) instances.
```

Comment:
How about:

> See `sh-init-list` for initialization/mutation, and `sh-assoc` for other operations on BashArray.

Reply:
This is essentially the same as another conversation https://github.com/oils-for-unix/oils/pull/2279#discussion_r1973990831, so I'll update them together. Let's discuss it in the other conversation.